File: src/Electrodomestico.java
public class Electrodomestico extends Producto {
        private String marca;
        private double voltaje;
        public Electrodomestico(String nombre, double precio, String fechaImportacion, int stock, String marca,double voltaje) {
            super(nombre, precio, fechaImportacion, stock);
            this.marca = marca;
            this.voltaje=voltaje;
        }

        @Override
        public String obtenerInformacion() {
            return super.obtenerInformacion() + ", marca:" + marca+ ", voltaje:"+voltaje;
        }
}

File: src/Electrodomestico.py

from Producto import Producto
class Electrodomestico(Producto):
    def __init__(self, nombre, precio, fecha_importacion, stock, marca, voltaje):
        super().__init__(nombre, precio, fecha_importacion, stock)
        self.__marca = marca
        self.__voltaje = voltaje

    def obtener_informacion(self):
        return f"{super().obtener_informacion()}, marca: {self.__marca}, voltaje: {self.__voltaje}"
